introduction/E_installation.md

```diff
@@ -45,6 +45,24 @@ We can get node.js from the [download page](https://nodejs.org/download/). When
 
 Mac OS X users can also install node.js via [homebrew](http://brew.sh/).
 
+#### Debian/Ubuntu users
+You might see an error that says something like:
```

Comment:
We can take the removed header from above and re-format this sentence to remove the "You", like this:

Debian/Ubuntu users might see an error that looks like this: